.docker/Pipfile

```diff
@@ -9,51 +9,62 @@ verify_ssl = true
 name = "pytorch-wheels"
 
 [packages]
-boto3 = "*"
-fastseg = "*"
-gdown = "*"
+addict = "==2.4.0"
```

Comment:
what's use of the package "addict"? why we need add it here?

Reply:
It is used in some notebooks e.g. 110 or 113, but this is a "weak" dependency. I assume it could be easily replaced in the future by ordinary Python dict.